src/javascript/crypto/e2e/openpgp/pgpmime/constants.js

```diff
@@ -56,10 +56,24 @@ e2e.openpgp.pgpmime.Constants.Mime = {
       'multipart/encrypted; protocol="application/pgp-encrypted"',
   NAME: 'name',
 
+  // Image specific Content Types. Case-insensitive.
+  IMAGE_GIF: 'image/gif',
+  IMAGE_JPEG: 'image/jpeg',
+  IMAGE_JPG: 'image/jpg',
+  IMAGE_PNG: 'image/png',
+  IMAGE_BMP: 'image/bmp',
+  IMAGE_WEBP: 'image/webp',
+  IMAGE_PJPEG: 'image/pjpeg',
+  IMAGE_TIFF: 'image/tiff',
+  IMAGE_SVG_XML: 'image/svg+xml',
+  IMAGE_VND_DJVU: 'image/vnd.djvu',
+
```

Comment:
My list was incomplete, thanks for pointing this out. 
I'll push a complete list of image subtypes, base on this list:
http://www.iana.org/assignments/media-types/media-types.xhtml#image